storybook/package.json

```diff
@@ -21,15 +21,15 @@
     "@storybook/addon-actions": "7.6.20",
     "@storybook/addon-essentials": "7.6.20",
     "@storybook/addons": "7.6.20",
-    "@storybook/api": "^7.6.20",
+    "@storybook/api": "7.6.20",
     "@storybook/components": "^7.6.20",
     "@storybook/core-events": "^7.6.20",
     "@storybook/manager-api": "7.6.20",
     "@storybook/theming": "^7.6.20",
     "@storybook/types": "7.6.20",
     "@storybook/web-components": "7.6.20",
     "@storybook/web-components-webpack5": "7.6.20",
-    "@whitespace/storybook-addon-html": "5.1.6",
+    "@whitespace/storybook-addon-html": "6.1.1",
```

Comment:
Deze update moet je ook even heroverwegen ivm een patch in onze code base op versie 5.1.6